.github/workflows/publish-release-tag.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Release Tag
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, e.g. v1.0, v20.15.10
+  workflow_dispatch:
+    # Allow manual trigger for non-standard cases.
+    # Unfortunately only allows branches are refs, currently (Jan 2021).
```

Comment:
This means you must select a branch to run the workflow on?
minor: should it be '.. allows branches as refs'?

Reply:
oops yeah.

Reply:
for manual (worker-dispatch) runs you can only select a branch, you cannot select a tag or input an arbitrary sha.

there's prob a way to work around that by doing a custom param that inputs into the checkout action but not taking that on for now.